migrator/migrations/20240805163854_circle_report.up.sql

```diff
@@ -0,0 +1,15 @@
+create table
+  "circle_report" (
```

Comment:
```suggestion
  "report" (
```

sepertinya better "report" saja biar lebih general
aku nanti bisa tambah relasi baru ke product jadinya

Reply:
ok, got it